apps/api/src/queries/term/term-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -93,23 +86,24 @@ const termCreated = new TestEventStream().andThen<TermCreated>({
     },
 });
 
-const termTranslated = termCreated.andThen<TermTranslated>({
-    type: 'TERM_TRANSLATED',
-    payload: {
-        translation: termTranslation,
-        languageCode: translationLanguage,
-    },
-});
+const dummyTerm = TermViewModel.fromTermCreated(termCreated.as(termCompositeId)[0] as TermCreated);
 
-const privateTermThatUserCanAccess = termTranslated.andThen<ResourceReadAccessGrantedToUser>({
-    type: 'RESOURCE_READ_ACCESS_GRANTED_TO_USER',
-    payload: {
-        userId: dummyQueryUserId,
-    },
+const targetTermView = clonePlainObjectWithOverrides(dummyTerm, {
+    isPublished: true,
+    contributions: [
+        {
+            id: dummyContributor.id,
+            fullName: dummyContributor.fullName.toString(),
+        },
+    ],
 });
 
-const termPublished = termTranslated.andThen<ResourcePublished>({
-    type: 'RESOURCE_PUBLISHED',
+const privateTermThatUserCanAccess = clonePlainObjectWithOverrides(dummyTerm, {
+    accessControlList: new AccessControlList({
+        allowedUserIds: [dummyQueryUserId],
+        allowedGroupIds: [],
+    }),
+    isPublished: false,
 });
 
 // TODO Add happy path cases for a prompt term
```

Comment:
This isn't super important because none of the logic in the query service or controller distinguishes prompt terms.